e2etests/test/authentication.js

```diff
@@ -181,4 +182,6 @@ module.exports = function authentication_tests(hubConnectionString) {
       });
     });
   });
-};
\ No newline at end of file
+};
+
+runTests();
```

Comment:
>runTests [](start = 0, length = 8)

I'd suggest just removing the function alltogether and just call describe directly

Reply:
(same for all scripts)

---
In reply to: [148617265](https://github.com/Azure/azure-iot-sdk-node/pull/133#discussion_r148617265) [](ancestors = 148617265)